condition/number_equal_to_test.go

```diff
@@ -134,6 +134,70 @@ var numberEqualToTests = []struct {
 		[]byte(`{"foo": 100, "bar": 200}`),
 		false,
 	},
+	{
+		"pass",
```

Comment:
Any tests that return false should be labeled as "fail" ("pass" is for tests that return true).